pkg/controller/persistentvolume/persistentvolume_recycler_controller.go

```diff
@@ -123,10 +205,32 @@ func (recycler *PersistentVolumeRecycler) reclaimVolume(pv *api.PersistentVolume
 			glog.Errorf(errMsg)
 			return fmt.Errorf(errMsg)
 		}
+	} else {
```

Comment:
logging in the `else` branch has proven to be valuable during testing, I'd like to leave as it is.